crates/bevy_reflect/src/serde/de.rs

```diff
@@ -280,7 +339,7 @@ impl<'a, 'de> DeserializeSeed<'de> for TypedReflectDeserializer<'a> {
                 let mut dynamic_struct = deserializer.deserialize_struct(
                     struct_info.name(),
                     // Field names are mainly just a hint, we don't necessarily need to store and pass that data
```

Comment:
Is this comment (and its sibling below) still accurate?

Reply:
Good catch! Yes I'll remove these as we now know at least `bincode` relies on them

Reply:
Fixed!